ouroboros-network-framework/src/Ouroboros/Network/Mux.hs

```diff
@@ -133,27 +133,39 @@ newtype OuroborosApplication (mode :: MuxMode) addr bytes m a b =
 
 
 -- |  There are three kinds of applications: warm, hot and established (ones
--- that run in for both warm and hot peers).
+-- that run in both warm and hot peers).
 --
 data ProtocolTemperature = Established | Warm | Hot
   deriving (Eq, Ord, Show)
 
-
--- | Singletons for 'AppKind'
+-- GR-FIXME[C3]: A name a little more specific, less misleading?
+--   The word "Temperature" makes one think Cold, Warm, or Hot.  But that's
+--   not the idea here.
+--   E.g.,
+--   data ProtocolRunsInTheTemperatureSet =
+--          WarmAndHot  / ProtocolsForWarmAndHot
+--        | WarmOnly    / ProtocolsForWarmOnly
+--        | HotOnly     / ProtocolsForHotOnly
+--   or, 
+--   data ProtocolCategories = ...
```

Comment:
We use `Established` for mini-protocols which runs for warm and hot
peers (called established in the outbound governor), `Warm` for warm
mini-protocols, `Hot` for hot mini-protocols.  This is the same naming
convention as the governor.

Status: refused.